README.md

```diff
@@ -98,6 +98,64 @@ Replace ``address`` with the actual adress and replace myhomebrew with the name
 
 If no result is returned, make sure to build with the ``-g`` or``-g3`` option to make sure psp-addr2line knowns the function names and locations.
 
+## Debugging with GDB
+
+Sometimes debugging with the built in instruction level debugger is not enough, this is where
+PSPLINK's GDB Server can come in handy.
+
+Let's get started:
+
+### Preparation
+
+Prepare a separate terminal for `usbhostfs_pc`, `pspsh` and `psp-gdb`. All of them are needed to point the directory location to your `.prx` and `.elf` files located or the file you need to debug.
+
+### 1. usbhostfs_pc
+
+Run `usbhostfs_pc` on your terminal dedicated for `usbhostfs_pc` and you will see the `waiting for device...` status and then start the PSPLINK app on your PSP and connect the USB cable, you will see the `connected to device` status, it means success. do not close this terminal after that.
+
+### 2. pspsh
+
+Run `pspsh` on your terminal dedicated for `pspsh` and you will see the `host0:/>`. After that you need to change again the directory where your files needed to debug. lastly, run `debug <file.prx>` and it will display something like this:
```

Comment:
You probably won't need to change directory. It will open where you ran usbhostfs_pc

Reply:
I would remove <> from the debug command for clarity